### PR TITLE
Fix: Ghosts element not ignored when assessing scenery or fountains

### PR DESCRIPTION
**Pull request by eezstreet**
Previously, if you had 39 pieces of scenery placed, and a ghost is present, it would count that ghost as the 40th piece of scenery, thus causing guests to think "Great scenery!"

### CODE REVIEW

**Review by duncanspumpkin — APPROVED** [review]
Add a changelog entry and bump network version

**Comment by eezstreet** [comment]
Changes made. Please reassess.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -82,6 +82,7 @@
 - Fix: [#18035] Favourited servers don’t get their online status updated.
 - Fix: [#18051] Visual glitch with Mine Ride's large unbanked turn (original bug).
 - Fix: [#18059] [Plugin] Width and height of custom window not changeable via script.
+- Fix: [#18122] Ghosts counted towards "Great scenery!" guest thought
```
Comment (Broxzier) [inline distribution/changelog.txt:85]: Explain the bug in present tense, and use fancy quotes: “text”

```suggestion
- Fix: [#18122] Ghosts count towards “Great scenery!” guest thought
```

File: src/openrct2/network/NetworkBase.cpp [hunk 2]
```diff
@@ -42,7 +42,7 @@
 // This string specifies which version of network stream current build uses.
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
-#define NETWORK_STREAM_VERSION "20"
+#define NETWORK_STREAM_VERSION "21"
```
Comment (duncanspumpkin) [inline src/openrct2/network/NetworkBase.cpp:45]: Rebase this is no longer the newest version

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**